### ISSUE TITLE
helm-install-traefik pod is failing

### ISSUE THREAD

**Issue by dlashua** [rancher/k3s#1347]
**Version:**

k3s version v1.17.0+k3s.1 (0f644650)

**Describe the bug**

The automatic job helm-install-traefik is failing. Pod is in CrashLoopBackOff Status.

**To Reproduce**

Just start k3s server

**Expected behavior**

For the job to complete

**Actual behavior**

The job doesn't complete.

**Additional context**

Pod Logs:

```
CHART=$(sed -e "s/%{KUBERNETES_API}%/${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/g" <<< "${CHART}")
set +v -x
+ cp /var/run/secrets/kubernetes.io/serviceaccount/ca.crt /usr/local/share/ca-certificates/
+ update-ca-certificates
WARNING: ca-certificates.crt does not contain exactly one certificate or CRL: skipping
+ + tiller export --listen=127.0.0.1:44134 HELM_HOST=127.0.0.1:44134
--storage=secret
+ HELM_HOST=127.0.0.1:44134
+ helm_v2 init --skip-refresh --client-only
[main] 2020/01/23 23:54:42 Starting Tiller v2.12.3 (tls=false)
[main] 2020/01/23 23:54:42 GRPC listening on 127.0.0.1:44134
[main] 2020/01/23 23:54:42 Probes listening on :44135
[main] 2020/01/23 23:54:42 Storage driver is Secret
[main] 2020/01/23 23:54:42 Max history per release is 0
Creating /root/.helm 
Creating /root/.helm/repository 
Creating /root/.helm/repository/cache 
Creating /root/.helm/repository/local 
Creating /root/.helm/plugins 
Creating /root/.helm/starters 
Creating /root/.helm/cache/archive 
Creating /root/.helm/repository/repositories.yaml 
Adding stable repo with URL: https://kubernetes-charts.storage.googleapis.com 
Adding local repo with URL: http://127.0.0.1:8879/charts 
$HELM_HOME has been configured at /root/.helm.
Not installing Tiller due to 'client-only' flag having been set
Happy Helming!
++ helm_v2 ls --all '^traefik$' --output json
++ jq -r '.Releases | length'
[storage] 2020/01/23 23:54:42 listing all releases with filter
+ EXIST=
+ '[' '' == 1 ']'
+ '[' '' == v2 ']'
+ helm_repo_init
+ grep -q -e 'https\?://'
+ echo 'chart path is a url, skipping repo update'
+ helm_v3 repo remove stable
chart path is a url, skipping repo update
Error: no repositories configured
+ true
+ return
+ helm_update install --set-string kubernetes.ingressEndpoint.useDefaultPublishedService=true --set-string metrics.prometheus.enabled=true --set-string rbac.enabled=true --set-string ssl.enabled=true
+ '[' helm_v3 == helm_v3 ']'
++ helm_v3 ls --all -f '^traefik$' --output json
++ jq -r '"\(.[0].app_version),\(.[0].status)"'
++ tr '[:upper:]' '[:lower:]'
+ LINE=1.7.19,failed
++ echo 1.7.19,failed
++ cut -f1 -d,
+ INSTALLED_VERSION=1.7.19
++ cut -f2 -d,
++ echo 1.7.19,failed
+ STATUS=failed
+ '[' -e /config/values.yaml ']'
+ '[' install = delete ']'
+ '[' -z 1.7.19 ']'
+ '[' -z '' ']'
+ '[' failed = deployed ']'
+ '[' failed = failed ']'
+ '[' helm_v3 == 'helm_v3]'
/usr/bin/entry: line 43: [: missing `]'
+ helm_v3 install --set-string kubernetes.ingressEndpoint.useDefaultPublishedService=true --set-string metrics.prometheus.enabled=true --set-string rbac.enabled=true --set-string ssl.enabled=true --purge traefik
Error: unknown flag: --purge
```

**Comment by erikwilson** [comment]
It looks like this is a dupe of #1332, should be fixed in v1.17.2-alpha3+k3s1

**Comment by dlashua** [comment]
Yep. A dupe. Sorry for the trouble.
